packages/p2p/src/stores/general-store.js

```diff
@@ -48,6 +48,8 @@ export default class GeneralStore extends BaseStore {
     parameters = null;
     payment_info = '';
     poi_status = null;
+    p2p_order_list_response_error = null;
+    p2p_order_list_response = [];
```

Comment:
should we not add these as observables?

Reply:
Good catch :+1:

Reply:
and the value for `p2p_order_list_response`. is it [] or {} since it has been destructured at  other places?

Reply:
yeah, i think its best to change it to {} since the return type is an object right @yaswanth-deriv

Reply:
dont forget to add these as observables